crux-core/src/crux/tx.clj

```diff
@@ -383,7 +383,15 @@
 
       (db/put-objects object-store docs)
 
-      (bus/send bus {::bus/event-type ::indexed-docs, :doc-ids (set (keys docs))})
+      (bus/send bus {::bus/event-type ::indexed-docs,
+                     :doc-ids (set (keys docs))
+                     :av-count (->> (vals docs)
+                                    (set)
+                                    (map count)
+                                    (reduce +))
+                     :bytes-indexed (->> (vals docs)
+                                         (map #(count (nippy/fast-freeze %)))
```

Comment:
it'll be expensive to re-serialise these just for the bus - let's see if we can re-use the existing serialisation